lib/Reader/reader.cpp

```diff
@@ -6994,6 +7001,7 @@ void ReaderBase::readBytesForFlowGraphNodeHelper(
     case ReaderBaseNS::CEE_LDC_I4_8:
     case ReaderBaseNS::CEE_LDC_I4_M1:
     LOAD_CONSTANT:
+      nop();
```

Comment:
For control flow I'd imagine that LLVM might optimize away "branch to next" even in debug codegen mode, so you'd lose the debug location if that happened.

If you're confident the only special case here (aside from branches) is `ldc` then just add a comment explaining why. If there are more special cases then I'd consider trying to do something more general.